pkg/config/config.go

```diff
@@ -229,11 +225,14 @@ func GetJSON(data interface{}) (string, error) {
 
 // OpenPiperFile provides functionality to retrieve configuration via file or http
 func OpenPiperFile(name string) (io.ReadCloser, error) {
-	//ToDo: support also https as source
 	if !strings.HasPrefix(name, "http") {
```

Comment:
Not part of this patch, and perhaps negligible, but you couldn't have a local file that starts with "http". More robust would be checking for "https://" and "http://".

Reply:
good point, I addressed that now ...